packages/nuxt/src/core/nuxt.ts

```diff
@@ -366,6 +366,15 @@ export async function loadNuxt (opts: LoadNuxtOptions): Promise<Nuxt> {
     options._modules.push('@nuxt/telemetry')
   }
 
+  // Nuxt DevTools is currently opt-in
+  if (options.devtools === true || (options.devtools && options.devtools.enabled === true)) {
```

Comment:
If provided an empty object, maybe a falsy check is safer?

```suggestion
  if (options.devtools === true || (options.devtools && options.devtools.enabled !== false)) {
```